site/content/contribute/containers/_index.md

```diff
@@ -0,0 +1,50 @@
+
+- [mattermost/mattermost-preview](https://hub.docker.com/r/mattermost/mattermost-preview) - This is a Docker image to install Mattermost in Preview Mode for exploring product functionality on a single machine using Docker. Preview image Docs: http://bit.ly/1W76riY. GitHub: https://github.com/mattermost/mattermost-docker-preview
+
+- [mattermost/platform](https://hub.docker.com/r/mattermost/platform) - Mirror of **mattermost/mattermost-preview**. This is a Docker image to install Mattermost in Preview Mode for exploring product functionality on a single machine using Docker. Preview image (mirror). Docs: http://bit.ly/1W76riY. GitHub: https://github.com/mattermost/mattermost-docker-preview
+
+- [mattermost/mattermost-loadtest](https://hub.docker.com/r/mattermost/mattermost-loadtest) - Image for the Load Test application. Tools for profiling Mattermost under heavy load. GitHub: https://github.com/mattermost/mattermost-load-test
+
+- [mattermost/sync-helpwanted-tickets](https://hub.docker.com/r/mattermost/sync-helpwanted-tickets) - For Internal usage. This iamge runs the sync with Jira tickets and GitHub Issues. To check the code please refer to https://github.com/mattermost/mattermost-utilities/tree/master/github_jira_tools
+
+- [mattermost/mattermost-build-server](https://hub.docker.com/r/mattermost/mattermost-build-server) - Image used to build Mattermost used in CI. To check the Docker file refer to https://github.com/mattermost/mattermost-server/blob/master/build/Dockerfile.buildenv
+
+- [mattermost/mattermost-build-webapp](https://hub.docker.com/r/mattermost/mattermost-build-webapp) - Image used to build Mattermost Webapp used in CI. To check the Docker file refer to https://github.com/mattermost/mattermost-webapp/blob/master/build/Dockerfile
+
+- [mattermost/podman](https://hub.docker.com/repository/docker/mattermost/podman) - Internal usage. Contain Podman to build/tag/push container images
+
+- [mattermost/mattermost-wait-for-dep](https://hub.docker.com/r/mattermost/mattermost-wait-for-dep) - Image used to wait for the other containers to start. Used in in CI and for local developement. Please refer to https://github.com/mattermost/mattermost-server/blob/master/docker-compose.yaml for more information.
+
+- [mattermost/mattermost-elasticsearch-docker](https://hub.docker.com/r/mattermost/mattermost-elasticsearch-docker) - Used in in CI and for local developement. Please refer to https://github.com/mattermost/mattermost-server/blob/master/docker-compose.yaml for more information.
+
+- [mattermost/webrtc](https://hub.docker.com/repository/docker/mattermost/webrtc) - DEPRECATED. Preview docker image of Mattermost WebRTC. Docs: http://bit.ly/2fVbWjU
```

Comment:
@enahum Any objections to deleting this image? It hasn't been updated since 2017 and we no longer support WebRTC natively.

Reply:
i support the idea to remove that